### ISSUE TITLE
Rebooting issues on the server

### ISSUE THREAD

**Issue by jfoclpf** [jfoclpf/autocosts#136]
When the server is restarted/rebooted internet is not immediately accessible and services get disabled by default, and are not enabled later on

```
There is no Internet connection
Services disabled: uber, social, database, googleCaptcha, googleAnalytics.
```

extracted from `pm2 logs`

```
jfolpf@contabo:/var/www/autocosts.prod$ pm2 logs 9
[TAILING] Tailing last 15 lines for [9] process (change the value with --lines option)
/home/jfolpf/.pm2/logs/autocosts.prod-error-9.log last 15 lines:
/home/jfolpf/.pm2/logs/autocosts.prod-out-9.log last 15 lines:
9|autocost | 30-03-2020: There is no Internet connection
9|autocost | 30-03-2020: Services disabled: 30-03-2020: uber, 30-03-2020: social, 30-03-2020: database, 30-03-2020: googleCaptcha, 30-03-2020: googleAnalytics.
9|autocost | 30-03-2020: Settings updated.
9|autocost | 30-03-2020: Listening on port 3028
9|autocost | 30-03-2020: To stop server press CTRL+C
9|autocost | 
9|autocost | 30-03-2020: *******************************************************************************
9|autocost | 30-03-2020: **                    The Car Costs Calculator                               **
9|autocost | 30-03-2020: **             can be now accessed on http://localhost:3028                  **
9|autocost | 30-03-2020: **                                                                           **
9|autocost | 30-03-2020: *******************************************************************************
9|autocost | 30-03-2020: 
9|autocost | Getting normalised costs from database table autocosts_prod->monthly_costs_normalized
9|autocost | 30-03-2020: World statistical data calculated
9|autocost | 30-03-2020: Statistical Data collected
```

**Comment by jfoclpf** [comment]
The commit 2b0153e54bb9eb0cf72853e55244235b5abe2e55 helped to solve the issue by attempting to connect to database every second until it is successfully, when the switch `database` is turned on

**Comment by jfoclpf** [comment]
Therefore solved with 2b0153e and 30a6fb3
